### ISSUE TITLE
Exchange rate calculator arrow svg on .currency select

### ISSUE THREAD

**Issue by nickBlack4** [bradtraversy/vanillawebprojects#20]
On Project 4, exchange rate calculator, the code for adding the svg as a background image is not necessary on Chrome on Ubuntu anyway.  I already had a default arrow like that and they clash if you add the SVG.

**Comment by bradtraversy** [comment]
Yeah, it shows by default for me on Windows but not Mac...not sure why this is not consistent across platforms. If this causes issues for anyone, just remove it. I will add a comment in the css
